src/app/job-routing/my-application/my-application.component.html

```diff
@@ -1,5 +1,5 @@
 <div class="container" *ngFor="let myApplication of Applications">
-    <a href="#">
+    <a routerLink="#">
```

Comment:
try not to make routerLink= "#"

Reply:
but since the route is not active yet, what should I do instead?